src/pages/student_detail/StudentDetail.tsx

```diff
@@ -36,81 +94,208 @@ const StudentDetail = () => {
-                  {payment.paymentDate && <img src={CheckIcon} width={30} />}
-                </PaymentInfo>
-              </Row>
-            ))}
-          </TableBody>
-        </PaymentsTable>
-      </PaymentsContainer>
+      {showConfirm && (
+        <ConfirmDialog
+          message={`¿Estás seguro de ${
+            student.status ? "dar de baja" : "dar de alta"
+          } a ${student.name} ${student.lastName}?`}
+          onConfirm={handleConfirm}
+          onCancel={() => setShowConfirm(false)}
+        />
+      )}
+      <HeaderContainer>
+        <TitleContainer>
+          <img
+            src={BackIcon}
+            alt="back-icon"
+            onClick={() => navigate(-1)}
+            style={{ cursor: "pointer" }}
+          />
+
+          <Title>DETALLE DEL ALUMNO</Title>
+        </TitleContainer>
+
+        <ButtonWrapper>
+          <Button
+            size="medium"
+            fontsize="large"
+            variant={student.status ? "warning" : "success"}
+            icon={
+              student.status ? (
+                <img src={DesactivateIcon} alt="desactivate-icon" />
+              ) : (
+                <img
+                  src={DesactivateIcon}
+                  alt="activate-icon"
+                  style={{ transform: "rotate(180deg)" }}
+                />
+              )
+            }
+            onClick={handleOpenConfirm}
+          >
+            {student.status ? "Dar de baja" : "Dar de alta"}
+          </Button>
+        </ButtonWrapper>
+      </HeaderContainer>
+
+      <StudentNameContainer>
+        <Title>
+          <IconStyles>
+            <img src={StudentIcon} alt="student-icon" />
+          </IconStyles>
+          {student.name} {student.lastName}
+        </Title>
+      </StudentNameContainer>
+      <InfoBoxesContainer>
+        <StudentData student={student} navigate={navigate} />
+        <PaymentData student={student} navigate={navigate} />
+      </InfoBoxesContainer>
     </MainContainer>
   );
 };
 
 export default StudentDetail;
+const StudentData = ({
+  student,
+  navigate,
+}: {
+  student: StudentDetailResponse;
+  navigate: (path: string) => void;
+}) => {
+  const info = [
+    {
+      title: "Nombre",
+      data: student.name,
+    },
+    {
+      title: "Apellido",
+      data: student.lastName,
+    },
+    {
+      title: "DNI",
+      data: student.dni,
+    },
+    {
+      title: "Fecha de ingreso",
+      data: student.admissionDate,
+    },
+    {
+      title: "Fecha de nacimiento",
+      data: student.birthday,
+    },
+    {
+      title: "Edad",
+      data: student.age,
+    },
+    {
+      title: "Número de teléfono",
+      data: student.cellphoneNumber,
+    },
+    {
+      title: "Patologías",
+      data: student.pathologies,
+    },
+  ];
+  return (
+    <StudentInfoContainer>
+      <HeaderBoxes>
+        <SubTitle>
+          <IconStyles>
+            <img src={StudentIcon} alt="student-icon" width={24} height={24} />
```

Comment:
```suggestion
            <img src={StudentIcon} alt="student-icon" width={20} height={20} />
```
Viendolo ya hecho capaz que este icono y el de info más chicos quedan un poco mejor, que pensas?